Automattic/wp-e2e-tests
# Add 'branches' target for live branch config to silence Slack

The live branches environment reports noise to Slack - we don't need this since you can read the results from a PR and a PR author won't be using the notifications channel anyway.

This adds a new TARGET to the config of BRANCHES and silences the notifications

<img width="329" alt="screen shot 2018-03-28 at 10 03 49 am" src="https://user-images.githubusercontent.com/128826/38001412-fad71794-326f-11e8-9033-661ae531c7d7.png">

**To Test**

1. `npm run decryptconfig`
2. make sure you can see the new config
3. `export TARGET=BRANCHES`
4. run some tests

@brbrr does this look right to you? All I need to do after merging this is add a TARGET value of BRANCHES to https://circleci.com/gh/Automattic/wp-e2e-tests-for-branches ?

PR: https://github.com/Automattic/wp-e2e-tests-for-branches/pull/22
